test/dockerfiles/synapse/log_config.yaml

```diff
@@ -17,6 +17,11 @@ handlers:
     # log to stdout, for easier use with 'docker logs'
     stream: 'ext://sys.stdout'
 
+
+# loggers:
+#     synapse.rest.media.v1.thumbnail_resource:
+#         level: DEBUG
```

Comment:
This isn't really related to this PR but was messing around with this on https://github.com/matrix-org/synapse/issues/13016 and https://github.com/matrix-org/synapse/issues/13039 and thought it was good reference whenever we want to add logging for something else down the line. Saves me from looking it up to copy-paste.